assignments/callbacks.js

```diff
@@ -21,27 +21,44 @@ const items = ['Pencil', 'Notebook', 'yo-yo', 'Gum'];
 
 */
 
-
+// getLength passes the length of the array into the callback.
 function getLength(arr, cb) {
-  return cb(arr[0]);
-  // getLength passes the length of the array into the callback.
+  return cb(arr.length);
 }
 
+getLength(items, function (items) {
+  console.log(items);
```

Comment:
Try separating out the anonymous function from the getLength parameters for practice. Invoking would look like this, getLength(items, someFunc);